libmscore/textedit.cpp

```diff
@@ -455,6 +456,13 @@ bool TextBase::edit(EditData& ed)
                         }
                   }
             }
+            if (ctrlPressed && altPressed) {
+                  switch (ed.key) {
+                        case Qt::Key_hyphen:
```

Comment:
Seems here a simple
'''
      if (ed.key == Qt::Key_hyphen)
            s = "\u203f"; // Unicode undertie
```
is more appropriate?
Unless you want to extend this to also handles short and long synalepha?

Reply:
I can't think of any universally usable shortcuts for those. And they are probably way to rare to be worth the effort.
So please make it that simple `if`.

Reply:
Done :)